lib/bigcommerce/connection.rb

```diff
@@ -34,24 +34,24 @@ def api_key=(api_key)
       @configuration[:api_key] = api_key
     end
 
-    def verify_peer=(verify)
+    def verify_ssl=(verify)
```

Comment:
Sure. I will make that change and update the README to reflect it. The only reason I suggested this change was because, it will be more inline with how `@configuration` is setup and also this is the way it was mentioned in README with the key `:verify_ssl`.